pkgs/development/python-modules/changefinder/fix_test_invocation.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/test/test.py b/test/test.py
+index 85a9f4e..9150a90 100644
+--- a/test/test.py
++++ b/test/test.py
+@@ -15,11 +15,13 @@ class TestChangeFinder():
+                                      np.random.rand(300)])
+ 
+     def test_changefinder(self):
++        self.setup()
```

Comment:
The [pytest documentation](https://docs.pytest.org/en/7.1.x/how-to/nose.html) suggests that it should automatically be calling the `setup()` method for nose compatibility. Do you have any idea why that’s not working? I’ll try to look into it if not.

Reply:
Didn't knew, not sure

Reply:
Ah, they removed the support: <https://docs.pytest.org/en/stable/deprecations.html#support-for-tests-written-for-nose>.

Could you rename `def setup(self):` (and `def teardown(self):` if any) to `def setup_method(self):`? That should make it just work without having to patch the individual tests.